src/lavinmq/queue/stream_queue_message_store.cr

```diff
@@ -8,11 +8,18 @@ module LavinMQ
       property max_length_bytes : Int64?
       property max_age : Time::Span | Time::MonthSpan | Nil
       getter last_offset : Int64
-      @segment_last_ts = Hash(UInt32, Int64).new(0i64) # used for max-age
+      @segment_last_ts = Hash(UInt32, Int64).new(0i64)     # used for max-age
+      @consumer_offset_positions = Hash(String, Int64).new # used for consumer offsets
+      @consumer_offsets : MFile
+      @consumer_offset_path : String
+      @consumer_offset_capacity = 32_768
```

Comment:
Why are these instance variables here? The same information can be accessed with `@consumer_offsets.path` and `@consumer_offsets.capacity`? 
```suggestion
```